src/app/about/page.tsx

```diff
@@ -1,7 +1,117 @@
+// "use client";
 import React from "react";
+import {Shield, Users, Target, Award} from "lucide-react";
+import "../globals.css";
+import Navbar from "@/components/Navbar";
 
-const About = () => {
-  return <div>About</div>;
+const page = () => {
```

Comment:
Retain the name of this component as "About" itself, lets have a consistent naming scheme since we are following it in other components as well

Reply:
Ok